be/src/common/config.h

```diff
@@ -413,6 +413,14 @@ CONF_mInt32(stream_load_record_batch_size, "50");
 CONF_Int32(stream_load_record_expire_time_secs, "28800");
 // time interval to clean expired stream load records
 CONF_mInt64(clean_stream_load_record_interval_secs, "1800");
+// use memory in stream load default
+CONF_Int64(stream_load_exec_mem_limit, "214748364"); // 2G
```

Comment:
CONF_mInt64(stream_load_exec_mem_limit, "214748364");
Maybe it's better to adjust this parameter dynamically？